sdk/typescript/src/verify/index.ts

```diff
@@ -29,6 +31,20 @@ export async function verifyPersonalMessage(
 ): Promise<PublicKey> {
```

Comment:
this method is not exposed in the sdk, i added the support here too for completeness.